Document Cron Builds (#578)
cirruslabs/cirrus-ci-docs: docs/guide/writing-tasks.md

    the variable by themselves and editing both the encrypted variable and repo-owner condition
    in the `.cirrus.yml` file.

## Cron Builds

> It is possible to configure invocations of re-occurring builds via a well-known Cron expressions. Cron builds can be

Reviewer: ```suggestion
It is possible to configure invocations of re-occurring builds via the well-known Cron expressions. Cron builds can be
```

> configured on a repository's settings page instead of configuring them in `.cirrus.yml`.

Reviewer: ```suggestion
configured in a repository's settings page, or in the repository's `.cirrus.yml`.
```

Author: There is no way to configure Cron Builds in `.cirrus.yml`

Reviewer: the way it was worded made it sound like it was possible

> configured on a repository's settings page instead of configuring them in `.cirrus.yml`.

Author: @RDIL how about this:

```suggestion
configured on a repository's settings page (not in `.cirrus.yml`).
```

Reviewer: that works

> It's possible to configure several cron builds with unique `names` which will be avaialbale via `CIRRUS_CRON` [environment variable](#environment-variables).

Reviewer: ```suggestion
It's possible to configure several cron builds with unique `names` which will be available via `CIRRUS_CRON` [environment variable](#environment-variables).
```

> Each cron build should specify *branch* to trigger new builds for and a cron expression compatible with Quarts. You can use

Reviewer 2: Quarts -> Quartz?

> Each cron build should specify *branch* to trigger new builds for and a cron expression compatible with Quarts. Please use

Reviewer: ```suggestion
Each cron build should specify *branch* to trigger new builds for and a cron expression compatible with Quarts. You can use
```

[this generator](https://www.freeformatter.com/cron-expression-generator-quartz.html) to generate/validate your expressions.

> **Note:** Cron Builds are using UTC timezone.

Reviewer: ```suggestion
**Note:** Cron Builds are timed with the UTC timezone.
```
